Arrangement_on_surface_2/demo/Arrangement_on_surface_2_earth/CMakeLists.txt

```diff
@@ -0,0 +1,130 @@
+# This is the CMake script for compiling a CGAL application.
+
+cmake_minimum_required(VERSION 3.1...3.23)
+project(Arrangement_on_surface_2_earth_Demo)
+
+if(NOT POLICY CMP0070 AND POLICY CMP0053)
+  # Only set CMP0053 to OLD with CMake<3.10, otherwise there is a warning.
+  cmake_policy(SET CMP0053 OLD)
+endif()
+
+if(POLICY CMP0071)
+  cmake_policy(SET CMP0071 NEW)
+endif()
+
+find_package(Qt6 QUIET COMPONENTS Core Gui OpenGL OpenGLWidgets Widgets Xml)
+find_package(CGAL COMPONENTS Qt6)
+
+if (NOT CGAL_FOUND OR NOT CGAL_Qt6_FOUND OR NOT Qt6_FOUND OR NOT Boost_FOUND)
+  if (NOT CGAL_FOUND)
+    set(MISSING_DEPS "the CGAL library, ${MISSING_DEPS}")
+  endif()
+  if (NOT CGAL_Qt6_FOUND)
+    set(MISSING_DEPS "the CGAL Qt6 component, ${MISSING_DEPS}")
+  endif()
+  if (NOT Qt6_FOUND)
+    set(MISSING_DEPS "the Qt6 library, ${MISSING_DEPS}")
+  endif()
+  if (NOT Boost_FOUND)
+    set(MISSING_DEPS "the Boost library, ${MISSING_DEPS}")
+  endif()
+
+  message(STATUS "This project requires ${MISSING_DEPS} and will not be compiled.")
+  return()
+endif()
+
+################################################################################
+include(FetchContent)
```

Comment:
@lrineau this is the first cmake script in CGAL where we use fetch content. Are we OK with using it?

Reply:
It actually seems to be responsible for the white columns in 6.0-Ic-194